.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+# A simple workflow to run tests using pytest against supported python versions
+
+name: Tests
+on: [pull_request]
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r tests/requirements.txt
+    - name: Build
+      run: |
+        python -m pip install .
```

Comment:
[question] Why is this command part of a separate `Build` step, rather than part of `Install dependencies`?

Reply:
Just as this installs our code, instead of dependencies of the test infra